src/core/Pattern_match.ml

```diff
@@ -78,6 +78,18 @@ type taint_trace_item = {
 type taint_trace = taint_trace_item list [@@deriving show, eq]
 type engine_kind = OSS | Pro [@@deriving show, eq]
 
+(* This type is used by postprocessors to report back the validity
+   of a finding. No_validator is currently also used when no
+   validation has yet occurred, which if that becomes confusing we
+   could adjust that, by adding another state. This corresponds to
```

Comment:
you can add instead a coupling comment like
coupling: with similar type defined in semgrep-interfaces/semgrep_output_v1.atd